docs/3.0/develop/inputs.mdx

```diff
@@ -767,17 +767,17 @@ python my_file_name greeter
 For example, with Prefect Cloud, you should see output like this:
 
 ```bash
-╭──────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ Your flow 'greeter' is being served and polling for scheduled runs!                              │
-│                                                                                                  │
-│ To trigger a run for this flow, use the following command:                                       │
-│                                                                                                  │
-│         $ prefect deployment run 'greeter/send-receive'                                          │
-│                                                                                                  │
-│ You can also run your flow via the Prefect UI:                                                   │
-│ https://app.prefect.cloud/account/...(a URL for your account)                                    │
-│                                                                                                  │
-╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
+______________________________________________________________________
+| Your flow 'greeter' is being served and polling for scheduled runs  |
+|                                                                    |
+| To trigger a run for this flow, use the following command:          |
+|                                                                    |
+|         $ prefect deployment run 'greeter/send-receive'            |
+|                                                                    |
+| You can also run your flow via the Prefect UI:                      |
+| https://app.prefect.cloud/account/...(a URL for your account)      |
+|                                                                    |
```

Comment:
Looks like there's a couple of `|` that are off by a space or two here.

Is it worthwhile to have CLI outputs like this in the docs, considering the effort required to keep it in sync with the actual output?

Reply:
Good catch. Updated. 

Good question. There are pros and cons to showing it. I don't have a strongly held preference.